### DISCUSSION TITLE
Welcome to AndroidMic Discussions!

### DISCUSSION THREAD

**Discussion by teamclouday** [teamclouday/AndroidMic#41] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dimwap** [comment]
How about.. What about using something other than Android, like an ESP32 with Wi-Fi? What's the data transfer format? Is there a description, or do I need to study the source code? (Although, what I'm saying is that AI makes it all easier.)
